src/pointer.rs

```diff
@@ -99,12 +99,16 @@ const fn is_valid_ptr(value: &str) -> bool {
 #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub struct Pointer(str);
 
+impl<'a> Default for &'a Pointer {
+    fn default() -> Self {
+        unsafe { &*("" as *const str as *const Self) }
+    }
+}
```

Comment:
I don't get the `'a` lifetime here, this is just `'static`, no?

Also why not use `Pointer::root()` in the body? This is effectively duplicating that code.

Reply:
Ugh, not sure why I did that - for whatever reason, I wasn't getting `Pointer::root` autocompletion so I just quickly spit something out.

Thanks for catching it. Changed it to:
```rust
impl Default for &'static Pointer {
    fn default() -> Self {
        Pointer::root()
    }
}
```